docs/asciidoc/modules/ROOT/pages/import/arrow.adoc

```diff
@@ -0,0 +1,121 @@
+[[load-arrow]]
+= Load and Import Arrow
+:description: This section describes procedures that can be used to import Apache Arrow data from web APIs or files.
+
+
+
+The following procedures allow you to read an Apache Arrow file exported via xref::export/arrow.adoc[apoc.export.arrow.* procedures].
+It could also potentially read other arrow files not created via the export procedures.
```

Comment:
It could also potentially read other Apache Arrow files that have not been created via the export procedures.